draft-ietf-rats-architecture.md

```diff
@@ -666,6 +649,49 @@ network Attester.
 In essence, an entity that combines more than one role also creates and
 consumes the corresponding conceptual messages as defined in this document.
 
+# Role Hosting and Composition
+
+NED VERSION
+
+The RATS architecture includes the definition of Roles (e.g. Attester, Verifier,
+Relying Party, Endorser) and conceptual messages (e.g., Evidence, Attestation Results,
+Endorsements, Appraisal Policies) that captures canonical attestation behaviors,
+that are common to a broad range of attestation-enabled systems. An entity that
+combines multiple Roles produces and consumes the associated Role Messages.
+
+The RATS architecture is not prescriptive about deployment configuration options
+of attestation-enabled systems, therefore the various Roles can be hosted on any
+participating entity. This implies, for a given entity, that multiple Roles could
+be co-resident so that the duties of multiple roles could be performed simultaneously.
+Nevertheless, the semantics of which Role Messages are inputs and outputs to a Role
+entity remains constant. As a result, the entity can participate as a constituent of the RATS architecture
+while flexibly accommodating the needs of various deployment architectures.
+
+Interactions between Roles do not necessarily require use of Internet protocols.
+They could, for example, use inter-process communication, local system buses,
+shared memory, hypervisors, IP-loopback devices or any communication path
+between the various environments that may exist on the entity that combines multiple Roles.
+
+The movement of Role Messages between locally hosted Roles is referred to as
+"local conveyance". Most importantly, the definition of local conveyance methods is
+out-of-scope for the RATS architecture.
+
+The following paragraph elaborates on an exemplary usage scenario:
```

Comment:
The above sentence is odd, why refer to "the following paragraph" with a header?

Reply:
Removing the colon would simplify this into a leading sentence?